looprpc/client.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.31.0
+// 	protoc-gen-go v1.33.0
```

Comment:
micro-nit (non-blocking): This should probably go into the previous commit.

Reply:
This is actually updated in this commit.